app-frontend/src/app/pages/projects/edit/edit.html

```diff
@@ -18,9 +18,16 @@ <h5 class="sidebar-title">Editing project</h5>
           </div>
         </li>
         <li>
-          <span class="label">Add/Remove Scenes</span>
-          <button class="btn btn-primary fixedwidth" type="button" ui-sref="projects.edit.scenes">
+          <span class="label">Scene Management</span>
+          <a class="btn btn-primary fixedwidth" ui-sref="projects.edit.scenes">
```

Comment:
https://davidwalsh.name/html5-buttons has a pretty good breakdown on when it makes sense to use each. 
I think it basically comes down to  whether we want to ability to disable the button. <a> attributes cannot be disabled, while <button> can, and it has a couple of other nice things such as :before and :after pseudo elements. 

This gets a bit murky when we consider that we have buttons / anchors which change the route, but we're using routing in some places to simplify state rather than to say "this is a separate page which has no relationship to the one you're coming from".  So it might make sense to use an anchor in the navbar, but not on the project edit view since each of the subviews is conceptually a part of the parent state.